src/app/core/mock-db.service.ts

```diff
@@ -2,13 +2,13 @@ import { Injectable } from '@angular/core';
 import { InMemoryDbService } from 'angular-in-memory-web-api';
 
 import { AngularNgbEksemplerData } from 'src/mock-db-data/library-angular/angular-ngb-eksempler.data';
-import { AngularFontAwesomeEksemplerData } from 'src/mock-db-data/library-angular/angular-font-awesome-eksempler.data';
+import { AngularFontAwesomeExamplesData } from 'src/mock-db-data/library-angular/angular-font-awesome-eksempler.data';
 
-import { CssSidemalerData } from 'src/mock-db-data/library-css/css-sidemaler.data';
-import { CssModulerData } from 'src/mock-db-data/library-css/css-moduler.data';
-import { CssKomponenterData } from 'src/mock-db-data/library-css/css-komponenter.data';
-import { CssFargerSkriftData } from 'src/mock-db-data/library-css/css-farger-skrift.data';
-import { CssIkonerData } from 'src/mock-db-data/library-css/css-ikoner.data';
+import { CssSidemalerData } from 'src/mock-db-data/library-css/css-page-templates.data';
```

Comment:
Should we go all the way while we're at it, even though PR are named "Files and folders..." ?

Reply:
Definitely